.github/workflows/run_cosi_test.yaml

```diff
@@ -1,5 +1,7 @@
 name: COSI integration Tests
-on: [push, pull_request, workflow_dispatch]
+# SHOULD BE RETURNED ONCE COSI IS BACK
+# on: [push, pull_request, workflow_dispatch]
+on: []
```

Comment:
@jackyalbo May we change it to `workflow_dispatch` meanwhile?
I receive errors in my email "No jobs were run".